Makefile

```diff
@@ -45,6 +46,18 @@ install-buf:
 install-go-test-coverage:
 	go install github.com/vladopajic/go-test-coverage/v2@v2.9.0
 
+.PHONY: install-abi-gen
+install-abi-gen:
+	go install github.com/ethereum/go-ethereum/cmd/abigen@v1.10.25
```

Comment:
It would be nice to eventually move all of the specific dependency versions for the `make install...` commands into `versions.json`